lib/puppetfactory.rb

```diff
@@ -82,6 +82,7 @@ def settings.puppetfactory
   get '/users/active/:username' do |username|
     session[:username] = username
     {"status" => "ok"}.to_json
+    redirect '/'
```

Comment:
Did you validate that this works for all browsers to redirect from an ajax API call?

Reply:
Yes.  Works for Firefox, Chrome, Safari, and IE11

Reply:
k. Line #84 is redundant then